website/docs/language/resources/ephemeral.mdx

```diff
@@ -0,0 +1,137 @@
+
+The Terraform language defines the following meta-arguments, which can be used
+with any ephemeral resource type to change the behavior of resources. These
+meta-arguments work the same way for resources, data sources, and ephemeral
+resources:
+
+- [`depends_on`, for specifying hidden dependencies](/terraform/language/meta-arguments/depends_on)
+- [`count`, for creating multiple resource instances according to a count](/terraform/language/meta-arguments/count)
+- [`for_each`, to create multiple instances according to a map, or set of strings](/terraform/language/meta-arguments/for_each)
+- [`provider`, for selecting a non-default provider configuration](/terraform/language/meta-arguments/resource-provider)
+- [`lifecycle`, for lifecycle customizations](/terraform/language/meta-arguments/lifecycle)
+
+Ephemeral resources to not support the `provisioner` meta-argument.
+
+## Ephemeral resource lifecycle
+
+The lifecycle of ephemeral resources is different resources and data sources.
+When provisioning ephemeral resources, Terraform will open the resource,
+optionally renew it one or more times, and then close it.
+
+1. When Terraform needs to access the ephemeral resource result, it will open
+the ephemeral resource. For example, opening a vault secret ephemeral resource
+means the Vault provider will obtain the lease and return the secret.
+
+1. If Terraform needs access to the ephemeral resource for longer than the
+expiration time enforced by the remote system, Terraform will ask the provider
+to renew it periodically. For example, renewing a Vault secret ephemeral
+resource means the Vault provider will call Vault's lease renewal API endpoint
+to extend the expiration time.
+
+1. Once Terraform no longer needs the ephemeral resource, Terraform will close
+it. This will happen once any providers that depend on the ephemeral resource
+have completed all of their work for the current phase. For example, closing a
+Vault secret ephemeral resource means the Vault provider will explicitly end the
+lease, allowing Vault to immediately revoke the associated credentials.
+
+Terraform will follow these lifecycle steps for each instance of an ephemeral
+resource in a given configuration.
+
+### Dependency graph
+
+Ephemeeral resources form nodes in Terraform's dependency graph which interact
+similarly to resources and data sources. For example, when a resource or data
+source depends on an attribute of an ephemeral resource, Terraform will
+automatically provision the ephemeral resource first.
+
+## Configuration model
+
+The following list outlines field hierarchy, language-specific data types, and
+requirements in the `moved` block. 
+
+- [`ephemeral`](#ephemeral): map
+  - [`attributes`](#ephemeral): string
+  - [`meta-arguments`](#ephemeral): string
+
+## Complete configuration
+
+An `ephemeral` block has the following form:
+
+```hcl
+ephemeral "<resource_type>" "<resource_name>" {
+  <attributes>
+  <meta-arguments>
+}
+```
+
+## Specification
```

Comment:
Maybe just remove this specification section for now? I understand it's tricky with a resource and I don't really think we need to figure this out for the beta